deploy/kube-config/influxdb/influxdb-grafana-controller.json

```diff
@@ -48,8 +48,20 @@
 			]
 		    },
 		    {
-			"image": "grafana/grafana:2.1.0",
-			"name": "grafana"
+			"image": "thuctnguyen/grafana:2.1.0",
```

Comment:
I built this image from the Dockerfile that is part of this PR. My suggestion is:

* docker pull thuctnguyen/grafana:2.1.0
* docker tag xyz kubernetes/grafana:2.1.0
* docker push kubernetes/grafana:2.1.0

I can then test with "kubernetes/grafana:2.1.0" and update the PR.